File: Converted/Models/CustomersContactTitle.vb
Imports System
Imports System.Collections.Generic

Namespace WindowsFormsApp1.Models
	Partial Public Class CustomersContactTitle
		Public Sub New()
			Customers = New HashSet(Of Customers)()
		End Sub

		Public Property ContactTitleId() As Integer
		Public Property Title() As String

		Public Overridable Property Customers() As ICollection(Of Customers)
	End Class
End Namespace

File: WindowsFormsApp1/Models/CustomersContactTitle.cs
using System;
using System.Collections.Generic;

namespace WindowsFormsApp1.Models
{
    public partial class CustomersContactTitle
    {
        public CustomersContactTitle()
        {
            Customers = new HashSet<Customers>();
        }

        public int ContactTitleId { get; set; }
        public string Title { get; set; }

        public virtual ICollection<Customers> Customers { get; set; }
    }
}
